languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml

```diff
@@ -3988,6 +3988,15 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
            MARCOAGPINTO inserted global antipatterns for all subrules on top using the logic (2-MAR-2023+)
       -->
 
+            <antipattern>
+                <token postag='RM' postag_regexp='no'/>
+                <token postag='DA.+' postag_regexp='yes'/>
```

Comment:
In your example, the `a` is a preposition, not an article.

Reply:
@p-goulart 

This is what I got:
![Screenshot 2023-09-30 at 13-31-22 Análise de Texto - LanguageTool](https://github.com/languagetool-org/languagetool/assets/5192600/e1020af1-31c4-4724-81ac-d79d576b6b8d)

Reply:
I'm not sayingn it's tagged as such, I am saying that's what it is. Using the `DA` tag to refer to a prepositional 'a' is dangerous:
- if the disambiguator ever gets fixed, this AP will stop working;
- it accepts 'as', which is legal.

This structure works **because** 'a' is a preposition. If it's not tagged correctly, I suggest adding a literal `a` to the antipattern. Or fixing the disambiguator :p

Reply:
Pedro, sorry, I was sleeping… 😋 I will fix it… just give me a few minutes.

ahhhh… I can't modify the disambiguator because I never learned how to, so I will just replace the `DA.+` with an `a`.

Reply:
@p-goulart 

It is fixed!

😋 😛 ❤️ 🤗